--- /tmp/Makefile.global.in	2013-09-10 20:36:59.000000000 -0700
+++ include/Makefile.global.in	2013-09-10 20:37:21.000000000 -0700
@@ -107,8 +107,8 @@
 # Add dynamic library entry point flags.
 ifdef LIBINIT
     ifneq (,$(findstring Darwin,$(uname)))
-	LDLIB	+= -init _$(LIBINIT)
+	LDLIB	+= -init _$(LIBINIT) -nostartfiles
     else
-	CFLAGS	+= -D$(LIBINIT)=_init
+	CFLAGS	+= -D$(LIBINIT)=_init -nostartfiles
     endif
 endif
